sycl/test-e2e/Graph/Explicit/kernel_bundle_spirv.cpp

```diff
@@ -0,0 +1,6 @@
+// RUN: %{build} -o %t.out
+// RUN: %if level_zero %{%{run} %t.out %S/../Inputs/Kernels/kernels.spv %}
```

Comment:
Does it make more sense to add `REQUIRES: level_zero` here instead of only guarding the execution line itself? There's not much point in building the test for unsupported backends, and then we can just add/change/remove the `REQUIRES` if/when we can enable this test for more backends.

We could also potentially add the standard extra `RUN` lines for L0 leaks/immediate command list testing here.

This comment also applies to the other new test.